docs/pages/tabs.md

```diff
@@ -215,7 +215,7 @@ Add the attribute `data-deep-link="true"` to a tabstrip to:
 </div>
 ```
 
-For example, <a target="_blank" href="#panel3d">http://example.com/#panel3d</a> will open the third tab panel at page load. This example will open a new browser tab and scroll you to the open tab.
+For example, <a target="_blank" href="#panel3d">//example.com/#panel3d</a> will open the third tab panel at page load. This example will open a new browser tab and scroll you to the open tab.
```

Comment:
Please readd the protocol and use https.

Reply:
example.com does not support https... will revert back to http.

Reply:
Woukld be now to me. See https://example.com/